source/Nuke.Common/Tools/NuGet/NuGet.Generated.cs

```diff
@@ -329,6 +329,102 @@ public static IEnumerable<(NuGetRestoreSettings Settings, IReadOnlyCollection<Ou
         /// <remarks>
         ///   <p>This is a <a href="http://www.nuke.build/docs/authoring-builds/cli-tools.html#fluent-apis">CLI wrapper with fluent API</a> that allows to modify the following arguments:</p>
         ///   <ul>
+        ///     <li><c>&lt;packageID&gt;</c> via <see cref="NuGetInstallSettings.PackageID"/></li>
```

Comment:
Wasn't sure whether to commit this file or not? It seems that it should be committed...